test/test_utils.py

```diff
@@ -389,61 +389,71 @@ def test_prepare_onnx_paddings(self):
         self.assertEqual(paddings, [0, 0, 3, 4, 1, 2])
 
     def test_check_onnx_broadcast(self):
-        #Case 1
+
+        def try_check_onnx_broadcast(dims1, dims2):
```

Comment:
nit: a nicer way would be to do `try_check_onnx_broadcast(dims1, dims2, expect_broadcast, expect_fail)` and put the asserts inside the function. Also, it's nicer to use `self.assertRaisesRegex` to ensure you're actually catching the errors you care about